RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs

```diff
@@ -0,0 +1,113 @@
+﻿using nl.SWEG.RPGWizardry.Player.Inventory;
+using nl.SWEG.RPGWizardry.Sorcery;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace nl.SWEG.RPGWizardry.ResearchData
+{
+    public class DataManager : MonoBehaviour
+    {
+        public PlayerInventory Inventory;
```

Comment:
Is Public set required on these variables? (Inventory can be found via PlayerManager)

Reply:
Playerinventory was unnecessary in the Datamanager, removed it.